angular-ngrx-scss/src/app/home/home.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
- [ ] delete the test file if there's nothing worth testing or `xdescribe` (if Karma-Jasmine) or `describe.skip(` (if Jest) if you plan on adding future tests to it.

Reply:
left this as `xdescribe` since the functionality won't exist until later in the project. can decide then if the test file should be improved or removed.